CHANGES.txt

```diff
@@ -1,24 +1,24 @@
 ### Login As 1.9.0 - February 18, 2015 ###
- * Updated for Elgg 1.9 (basically just tested it still works and it does)
- * Added German translations
- * Added Finnish translations
+ * Updated for Elgg 1.9 (basically just tested it still works and it does),
+ * Added German translations,
+ * Added Finnish translations.
```

Comment:
This is using list syntax so there's no need for trailing commas or dots. This file should be converted to markdown syntax anyway, and the commas would look weird at the end of each row.